.clang-format

```diff
@@ -0,0 +1,174 @@
+---
+Language: Cpp
+# BasedOnStyle:  LLVM
```

Comment:
Reading the documentation, we could actually activate this entry and then we do not need most of the other entries. Instead we can specify those values that we want to change. It seems to me that would be more understandable.

Reply:
Fixed

Reply:
What I meant to say is that now it should be possible to remove all the other entries in the `.clang-format` file, no?
Except the changed `ColumnLimit`. And whatever else we want to customize.

Reply:
Hmm, understood, I think done